org.libvips.nip4.json

```diff
@@ -0,0 +1,478 @@
+            "config-opts" : [
+                    "-DCMAKE_INSTALL_LIBDIR=lib",
+                    "-DBUILD_CPP_TESTS=OFF",
+                    "-DBUILD_GTK_TESTS=OFF",
+                    "-DBUILD_MANUAL_TESTS=OFF",
+                    "-DBUILD_QT5_TESTS=OFF",
+                    "-DBUILD_QT6_TESTS=OFF",
+                    "-DENABLE_LIBTIFF=ON",
+                    "-DENABLE_LIBPNG=ON",
+                    "-DENABLE_GLIB=ON",
+                    "-DENABLE_LIBOPENJPEG='openjpeg2'",
+                    "-DENABLE_DCTDECODER='libjpeg'",
+                    "-DENABLE_BOOST=OFF",
+                    "-DENABLE_UTILS=OFF",
+                    "-DENABLE_QT5=OFF",
+                    "-DENABLE_QT6=OFF",
+                    "-DENABLE_CPP=OFF"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://poppler.freedesktop.org/poppler-24.10.0.tar.xz",
+
+                    "sha256" : "58ed1849810f04a10b37c7ff6f3e411845c8a57d731d599d0045acc7a7fff09d"
+                }
+            ]
+        },
+
+        {
+            "name" : "imath",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "build-options" : {
+                "config-opts" : [
+                    "-DCMAKE_INSTALL_LIBDIR=lib"
+                ]
+            },
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/AcademySoftwareFoundation/Imath.git",
+                    "tag" : "v3.1.12",
+                    "commit" : "c0396a055a01bc537d32f435aee11a9b7ed6f0b5"
+                }
+            ]
+        },
+
+        {
+            "name" : "openexr",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "build-options" : {
+                "config-opts" : [
+                    "-DCMAKE_INSTALL_LIBDIR=lib"
+                ]
+            },
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/AcademySoftwareFoundation/openexr.git",
+                    "tag" : "v3.3.1",
+                    "commit" : "f723c3940fff287c0a26b425b90a8e764823bfd4"
+                }
+            ]
+        },
+
+        {
+            "name" : "uthash",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -D include/* /app/include"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/troydhanson/uthash.git",
+                    "commit" : "ca98384ce7f30beb216f9a0bc88a3b4340ead729"
+                }
+            ]
+        },
+
+        {
+            "name" : "libdicom",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release",
+                "--libdir=lib",
```

Comment:
you shouldn't need to set meson lidir either. same as CMake.

Reply:
done!